examples/cra-kitchen-sink/src/stories/image-diff.stories.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { setOptions } from '@storybook/addon-options';
+import { action } from '@storybook/addon-actions';
+import { withNotes, WithNotes } from '@storybook/addon-notes';
+import centered from '@storybook/addon-centered';
+import { withInfo } from '@storybook/addon-info';
+import { Button } from '@storybook/react/demo';
+
+import Container from './Container';
+
+const InfoButton = () => (
+  <span
+    style={{
+      fontFamily: 'sans-serif',
+      fontSize: 12,
+      textDecoration: 'none',
+      background: 'rgb(34, 136, 204)',
+      color: 'rgb(255, 255, 255)',
+      padding: '5px 15px',
+      margin: 10,
+      borderRadius: '0px 0px 0px 5px',
+    }}
+  >
+    {' '}
+    Show Info{' '}
+  </span>
+);
+
+storiesOf('Button - for-image-snapshot', module)
```

Comment:
Can you please follow the current naming convention we have?

`storiesOf('Addon Storyshots.Button...')`

Reply:
I need a pattern to match to avoid generating all images. 

WDYT about `Addon Storyshots.Button.ImageSnapshot`  where `ImageSnapshot` is the pattern I'm matching ?

Reply:
what's wrong with matching that entire string? is it because the entire kind name isn't available to you because of the hierarchy tree?